src/Model/Post.php

```diff
@@ -607,6 +607,17 @@ public static function selectOriginalForUser(int $uid, array $selected = [], arr
 		return self::selectFirstForUser($uid, $original_selected, ['uri-id' => $result['thr-parent-id'], 'uid' => [0, $uid]], $params);
 	}
 
+	/**
+	 * Retrieves a list of up to 100 unseen posts for a user
+	 * @param int $uid User ID
+	 * @return array Array containing the URI IDs of unseen posts
+	 */
+	public static function getUnseenPosts(int $uid): array
```

Comment:
Will it be statically 100? Maybe you can therefore name it

```suggestion
	public static function getFirstHundredUnseenPosts(int $uid): array
```

Just an idea, it's not a must have

Reply:
I would like to keep the `100` just in the description. Who knows? Possibly it does make sense in the future to change that value.